ci/dependency_update.sh

```diff
@@ -4,6 +4,6 @@ set -evx
 
 . ci/bundle_install.sh
 
-bundle exec rake dependencies
+bundle exec rake dependencies_ci
```

Comment:
Yes! :) dependencies is for humans, which basically triggers ci/dependency_update.sh now. If you run `rake dependencies` (now _ci) it typically won't work on your local machine because of the multi-os-ness of the gem deps